frontend/src/components/User/User.js

```diff
@@ -15,58 +17,38 @@ export const User = props => {
         event.preventDefault();
         axios.post(global.config.BACKEND_URL + "/account/following/newFollower/" + props.username)
             .then(
-                (response) => {
-                    setFollowing(true);
-                },
+                (response) => { setFollowing(true); },
                 (error) => {
-                    if (error.response)
-                    {
-                        setErrorMsg(error.response.data.message);
-                    } 
-                    else
-                    {
-                        setErrorMsg("An unknown error occurred");
-                    }
+                    if (error.response) { setErrorMsg(error.response.data.message); }
+                    else { setErrorMsg("An unknown error occurred"); }
                 }
             )
     };
 
     const unfollow = (event) => {
         event.preventDefault();
-        axios.delete(global.config.BACKEND_URL + "/account/following/followerRemoval/"+ props.username)
+        axios.delete(global.config.BACKEND_URL + "/account/following/followerRemoval/" + props.username)
```

Comment:
This should be switch to a `then-catch` block.

Reply:
This entire block was actually deleted/removed from this file when @daniela-venuta did her PR concerning the FollowButton and has since then been changed to a `then-catch` block 👍